stable/pipelines/templates/metadata/grpc-deployment.yaml

```diff
@@ -2,21 +2,21 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: metadata
+  name: metadata-grpc-deployment
```

Comment:
is the `-deployment` suffix to comply with kubeflow/pipelines naming or your doing?
If to comply 👍 
else - I think we can let this go (putting the resource type in the name here, it's sometimes done but IIRC on deployments it's not)

Reply:
it's to comply with what they did 😄